src/main/java/org/checkerframework/specimin/MethodPrunerVisitor.java

```diff
@@ -106,4 +118,61 @@ public Visitable visit(ConstructorDeclaration constructorDecl, Void p) {
       return constructorDecl;
     }
   }
+
+  @Override
+  public Visitable visit(FieldDeclaration fieldDecl, Void p) {
+    String classFullName = fieldDecl.resolve().declaringType().getQualifiedName();
+    boolean isFinal = fieldDecl.isFinal();
+    Iterator<VariableDeclarator> iterator = fieldDecl.getVariables().iterator();
+    while (iterator.hasNext()) {
+      VariableDeclarator varDecl = iterator.next();
+      String varFullName = classFullName + "#" + varDecl.getNameAsString();
+
+      if (methodsToEmpty.contains(varFullName)) {
```

Comment:
I think the name `methodsToEmpty` is out-of-date given that this PR deals with fields. We should change `method` in the name of `methodsToEmpty` to `member` - i.e., it should be `membersToEmpty`, since "members" of a class include both its fields and its methods.

There is a similar problem with the class, which is named `MethodPrunerVisitor`, but has now been augmented to also prune fields. I'd rename it `MemberPrunerVisitor` or maybe just `PrunerVisitor`; the class Javadoc will also need to change appropriately.